### PR TITLE
fix: add siddhi-244

### PR DESCRIPTION
**Pull request by siddhi-244**

### CODE REVIEW

File: public/data/SiddhiBhanushali.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/siddhi-244",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SiddhiB6",
+      "icon": "twitter"
+    }
+  ]
```
Comment (schmelto) [inline public/data/SiddhiBhanushali.json:16]:
```suggestion
  ]
}
```

File: public/data/SiddhiBhanushali.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Siddhi Bhanushali",
+  "bio": "Web development & open source enthusiast",
+  "avatar": "https://avatars.githubusercontent.com/u/69195262?v=4",
```
Comment (schmelto) [inline public/data/SiddhiBhanushali.json:4]:
```suggestion
  "avatar": "https://github.com/siddhi-244.png",
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by siddhi-244** [comment]
Done !

**Review by krishguptadev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

File: public/data/SiddhiBhanushali.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Siddhi Bhanushali",
+  "bio": "Web development & open source enthusiast",
```
Comment (Panquesito7) [inline public/data/SiddhiBhanushali.json:3]: It should be `Web developer`.
```suggestion
  "bio": "Web developer & open source enthusiast",
```

Comment (nhcarrigan) [reply public/data/SiddhiBhanushali.json:3]: Maybe they're saying they're a web development enthusiast?

**Pull request merged**